.travis-build.sh

```diff
@@ -7,6 +7,15 @@ bootstrap_header() {
   echo -e "${ANSI_YELLOW}$1${ANSI_RESET}"
 }
 
+travis_fold() {
```

Comment:
AFAIR the reason for not sourcing was a problem with `set -e` which terminated the whole build including the `after_failure` clause in the travis script. Running it as `bash script` and setting set -e fixed that.

Without `set -e` you have to check each command so that the build fails if one of the steps fails (e.g. the problem you noticed that the test builds do not show that something is wrong...).

I will try to get a fix for both problems... Going back to a "travis.yml only" setup would be my preference... IMO one big travis.yml is better than a travis.yml which links to a big script. Especially if the amount of code is reduced because we don't need to reimplement most of travis' r support?